cmd/athenz-sia/main.go

```diff
@@ -84,6 +87,21 @@ func main() {
 		return
 	}
 
+	// show version and build date in metrics
+	if !idConfig.Init && idConfig.MetricsServerAddr != "" {
+		promauto.NewGaugeFunc(prometheus.GaugeOpts{
+			Name: "build_info",
+			Help: "A metric with a constant '1' value labeled with app name, version, build date",
+			ConstLabels: prometheus.Labels{
+				"app_name":   filepath.Base(os.Args[0]),
+				"version":    VERSION,
+				"build_date": BUILD_DATE,
+			},
+		}, func() float64 {
+			return float64(1)
+		})
+	}
```

Comment:
Am I correct that you actually do not need the condition `if !idConfig.Init && idConfig.MetricsServerAddr != ""` as it is just registering the metrics?

Reply:
Rephrase comments and help.
using `1` is a common practice in Prometheus and therefore you do not have to specifically define what `1` means here. Also since default official ones start with `Indicate` (verb) let's stick with the same verb.


```suggestion
	// register a metric to display the application's app_name, version and build_date
	if !idConfig.Init && idConfig.MetricsServerAddr != "" {
		promauto.NewGaugeFunc(prometheus.GaugeOpts{
			Name: "build_info",
			Help: "Indicates the application name, build version and date",
			ConstLabels: prometheus.Labels{
				"app_name":   filepath.Base(os.Args[0]),
				"version":    VERSION,
				"build_date": BUILD_DATE,
			},
		}, func() float64 {
			return float64(1)
		})
	}
```

Other log for example:
```sh
# HELP x509_cert_expires_in_seconds Indicates the remaining time before the certificate's not after timestamp
# TYPE x509_cert_expires_in_seconds gauge
x509_cert_expires_in_seconds{filename="ca.cert.pem" ...}

```

Reply:
comment fixed: https://github.com/AthenZ/k8s-athenz-sia/pull/88/commits/fe18881d7f3b1de89feabb49100fa2de09b8af06